src/tools/perf/api/libperf.h

```diff
@@ -117,7 +117,7 @@ typedef struct ucx_perf_result {
     double                  elapsed_time;
     ucx_perf_counter_t      bytes;
     struct {
-        double              typical;
+        double              percentile;
         double              moment_average; /* Average since last report */
         double              total_average;  /* Average of the whole test */
     }
```

Comment:
what are the units?

Reply:
Microseconds, should I add a comment behind this declaration?